src/plugins/vis_type_vislib/public/vislib/lib/chart_title.test.js

```diff
@@ -19,11 +19,15 @@
 
 import d3 from 'd3';
 import _ from 'lodash';
-import expect from '@kbn/expect';
+import {
+  mockHTMLElementClientSizes,
+  setSVGElementGetBBox,
+  setSVGElementGetComputedTextLength,
```

Comment:
depends on #70066
@VladLasitsa when #70066 will be merged please modify  `../../../../../test_utils/public/helpers` -> `../../../../../test_utils/public`

Reply:
done